patroni/dcs/consul.py

```diff
@@ -233,6 +233,7 @@ def __init__(self, config):
         if self._register_service:
             self._set_service_name()
         self._service_check_interval = config.get('service_check_interval', '5s')
+        self._service_check_tls_server_name = config.get('service_check_tls_server_name', "")
```

Comment:
the better place for it is the `reload_config()` method, so one could change it on the fly.

Reply:
I tested moving these values to reload_config, changing them has no effect on reload - for that, touch_member would have to be called. I don't know patroni well enough to understand where I would add that and what the consequences would be.

Reply:
Maybe you are right, supporting reloading `service_check_tls_server_name` on the fly isn't really required.